docs/usage.md

```diff
@@ -0,0 +1,120 @@
+# Usage
+
+Aleph main features are available through the web interface. Data loading
+and other maintenance operations are provided via command line tools.
+In order to understand the concepts and how it works, please refer to
+the chapters bellow.
+
+## Use cases
+
+Below are some of common use cases Aleph was designed to cover. To learn more
+about the needs of such a tool, please refer to the notes about the
+[user needs in investigative journalism](http://www.influencemapping.org/workshop/user_needs.html).
+A lot more resources are also available on the
+[Technology for Investigative Journalism Conference](http://www.influencemapping.org/workshop/) webpage.
+
+There's also a [glossary](glossary.md) describing the keywords used in Aleph.
+
+Consider some common use-cases like:
```

Comment:
I like that you added the use cases into the usage document, that really helps make it more plastic